pkg/controller/dataimportcron-controller.go

```diff
@@ -311,13 +313,36 @@ func (r *DataImportCronReconciler) update(ctx context.Context, dataImportCron *c
 	dataImportCronCopy := dataImportCron.DeepCopy()
 	imports := dataImportCron.Status.CurrentImports
 	importSucceeded := false
+
+	dataVolume := dataImportCron.Spec.Template
+	explicitScName := getStorageClassFromTemplate(&dataVolume)
+	dvStorageClass, err := cc.GetStorageClassByName(r.client, explicitScName)
+	if err != nil {
+		return res, err
```

Comment:
@arnongilboa having second thoughts about this wrt to https://github.com/kubevirt/containerized-data-importer/pull/2683. maybe better to take the sc name from the created pvc?

Reply:
If you want to support the case the DV SC is not found yet, or it's using the default which is not set yet, taking `sc name from the created pvc` is sure the safest, but does it fit your code?

Reply:
yeah pvc may be racy, I went with just proceeding with DV creation as before.
added a unit test for this behavior